herddb-core/src/test/java/herddb/core/CreateTableTest.java

```diff
@@ -63,4 +63,50 @@ public void createTable1() throws Exception {
         }
 
     }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testWithNullPrimaryKey() throws Exception {
```

Comment:
Not sure about the value of this test.
For backward compatibility we have to allow PKs which are defined with NULLABLE column types

Reply:
or is this test about using a DOUBLE column ?

Reply:
This test is to ensure we throw exception when primary key is chosen as Double.

Reply:
please rename accordingly

Reply:
Done.